src/actions/index.js

```diff
@@ -646,6 +649,22 @@ export function editTagDescription (tag, description) {
   return {type: EDIT_TAG_DESCRIPTION, payload: {tag, description}}
 }
 
+export function showTagPopover (tagName, slug, position, anchorWidth) {
+  return {type: SHOW_TAG_POPOVER, payload: {tagName, slug, position, anchorWidth}}
+}
+
+export function hideTagPopover () {
+  return {type: HIDE_TAG_POPOVER}
+}
+
+export function fetchTagSummary (tagName, id) {
+  return {
+    type: FETCH_TAG_SUMMARY,
+    payload: {api: true, path: `/noo/community/${id}/tag/${tagName}/summary`},
+    meta: {tagName, id}
```

Comment:
It actually is cached, in that, the old data is already in the store and displays instantly, and then this is called and if there's any change to the data it's updated when this returns